CHANGELOG.md

```diff
@@ -2,10 +2,10 @@
 
 All notable changes to the LaunchDarkly git-flag-parser will be documented in this file. This project adheres to [Semantic Versioning](http://semver.org).
 
-## [0.1.1] - 2019-01-11
```

Comment:
intentionally removed. the convention I like is to just keep building up the changelog for whats in master, but not add a version or date until we're ready to release.